src/main/java/org/folio/orders/utils/HelperUtils.java

```diff
@@ -1159,4 +1159,8 @@ public static JsonObject verifyProtectedFieldsChanged(List<String> protectedFiel
       .map(HelperUtils::convertToPoLine)
       .collect(toList());
   }
+
+  public static boolean isProductIdsExist(CompositePoLine compPOL) {
+    return compPOL.getDetails() != null && CollectionUtils.isNotEmpty(compPOL.getDetails().getProductIds());
```

Comment:
It may make sense to foresee the case when ProductIdType doesn't exist in request because this field isn't required in schema?

Reply:
I think we'll have to make `productIdType` required,since this field is used to add identifiers to the instance and it is required. 
https://github.com/folio-org/mod-orders/blob/cfd04c3cfd4248350b3b6bd851a89b90bd2088ca/src/main/java/org/folio/rest/impl/InventoryHelper.java#L485
https://github.com/folio-org/mod-inventory-storage/blob/9405f40053e24ed79912ef0c583e05bce59fa017/ramls/instance.json#L81-L82

Reply:
I don't think we can make productIdType required because we have scenarios where inventory interaction is not required

Reply:
Forgot to link a story [MODORDSTOR-115](https://issues.folio.org/browse/MODORDSTOR-115) that I created to handle this. There are some clarifications from the POs in the comments.